addons/a11y/src/components/ColorBlindness.tsx

```diff
@@ -83,7 +93,7 @@ class ColorBlindness extends Component {
         placement="top"
         trigger="click"
         tooltipShown={expanded}
-        onVisibilityChange={s => this.setState({ expanded: s })}
+        onVisibilityChange={(s: boolean) => this.setState({ expanded: s })}
```

Comment:
To prevent [performance issue](https://reactjs.org/docs/faq-functions.html#arrow-function-in-render) down the road, please put it into a [class property](https://reactjs.org/docs/faq-functions.html#class-properties-stage-3-proposal).